app/gateway/upgrade/lts-upgrade.md

```diff
@@ -8,15 +8,200 @@ products:
+
+### Performing the upgrade
+
+The actual execution of the upgrade depends on type of deployment you have with {{site.base_gateway}}. 
+In this part of the upgrade journey, you will use the strategy you determined during the preparation phase.
+
+1. Execute your chosen upgrade strategy on dev.
+2. Move from dev to prod.
+3. Smoke test.
+4. Wrap up the upgrade or roll back and try again.
+
+Now, let's move on to preparation, starting with your backup options.
+
+## Preparation: Choose a backup strategy
+
+{% include_cached /upgrade/backup.md %}
+
+## Preparation: Choose an upgrade strategy based on deployment mode
+
+Upgrade strategies introduced in this section are generic and may or may not fit in with your deployment environment. 
+
+Choose your deployment mode:
+* [Traditional](#traditional-mode)
+* [DB-less](#db-less-mode)
+* [Hybrid](#hybrid-mode)
+
+Here's a flowchart that breaks down how the decision process works:
+
+{% include_cached /upgrade/flow.md %}
+
+See the following sections for breakdowns of each strategy.
+
+### Traditional mode
+
+{% include_cached /upgrade/traditional.md %}
+
+#### Dual-cluster upgrade
+
+Upgrading {{site.base_gateway}} from one LTS version to another LTS version with zero downtime can be achieved through a [dual-cluster upgrade strategy](/gateway/upgrade/dual-cluster/). 
+This approach involves setting up a new cluster running the upgraded version of {{site.base_gateway}} alongside the existing cluster running the current version.
+
+At a high level, the process typically involves the following steps:
+
+1. **Provisioning a same-size deployment**: You need to ensure that the new cluster, which will run the upgraded version of {{site.base_gateway}}, has the same capacity and resources as the existing cluster. This ensures that both clusters can handle the same amount of traffic and workload.
+
+2. **Setting up dual-cluster deployment**: Once the new cluster is provisioned, you can start deploying your APIs and configurations to both clusters simultaneously. The dual cluster deployment allows both the old and new clusters to coexist and process requests in parallel.
+
+3. **Data synchronization**:  During the dual cluster deployment, data synchronization is crucial to ensure that both clusters have the same data. This can involve migrating data from the old cluster to the new one or setting up a shared data storage solution to keep both clusters in sync. Import the database from the old cluster to the new cluster by using a snapshot or `pg_restore`.
+
+4. **Traffic rerouting**: As the new cluster is running alongside the old one, you can start gradually routing incoming traffic to the new cluster. This process can be done gradually or through a controlled switchover mechanism to minimize any impact on users. This can be achieved by any load balancer, like DNS, Nginx, F5, or even a {{site.base_gateway}} node with Canary plugin enabled.
```

Comment:
```suggestion
1. **Traffic rerouting**: As the new cluster is running alongside the old one, you can start gradually routing incoming traffic to the new cluster. This process can be done gradually or through a controlled switchover mechanism to minimize any impact on users. This can be achieved by any load balancer, like DNS, Nginx, F5, or even a {{site.base_gateway}} node with Canary plugin enabled.
```